plugins/catalog-react/src/components/UserListPicker/UserListPicker.tsx

```diff
@@ -122,11 +122,18 @@ function getFilterGroups(
 export type UserListPickerProps = {
   initialFilter?: UserListFilterKind;
   availableFilters?: UserListFilterKind[];
+  hidden?: boolean;
+  resetFiltersWhenNoResultsFound?: boolean;
 };
 
 /** @public */
 export const UserListPicker = (props: UserListPickerProps) => {
-  const { initialFilter, availableFilters } = props;
+  const {
+    initialFilter,
+    availableFilters,
+    hidden,
+    resetFiltersWhenNoResultsFound = true,
```

Comment:
I'd rather flip the logic of this one around so that it becomes a boolean flag that you set to true to change the behavior, it avoids an awkward `flag={false}` prop instead of just `flag`.

Perhaps `alwaysKeepFilters` or something like that?